packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/wdio-reporters.3/FileOutputReporter.mdx

```diff
@@ -4,7 +4,7 @@ Wdio File Output Reporter is a reporter that logs wdio test output to separate f
 
```

Comment:
I don't see the File Output Reporter in [terra-ui](https://engineering.cerner.com/terra-ui/dev_tools/cerner-terra-toolkit-docs/terra-functional-testing/wdio-reporters/spec-reporter). Maybe the `FileOutputReporter.mdx` file name doesn't have `.tool` in it as in `FileOutputReporter.tool.mdx`